### DISCUSSION TITLE
Dark Mode - how to disable

### DISCUSSION THREAD

**Discussion by peroxovy** [Power-Components/livewire-powergrid#1056] [Q&A]
Hello, Is there possibility to disable dark mode using powergrid?

I've tried several times to delete styles from `vendor` but I think it's not good idea. 

Any idea how to disable that completely?

**Comment by luanfreitasdev** [comment]
I don't know if I understand correctly, why give more information?

**Comment by kosta006** [comment]
I have followed the instructions and simply removed the darkMode: 'class', from the tailwind.,config.js file but dark mode styles are still being applied in some areas for instance the withDeleted filter options. Does anyone know how to disabled ark mode completely?
